docs/data_access.rst

```diff
@@ -115,3 +115,24 @@ on the collection and the catalog that is used by it. This is not a problem caus
 standardization between catalogs of EO data.
 
 
+Handling large vector data
+#######################
+
+Handling large volumes of data is crucial for making decisions, improving processes and having an efficient solution. 
+However, if you attempt to execute a process on significantly large data, you would probably encounter a 'Request Entity Too Large' 
+error because your 'POST' request was too large for the system to handle. In order to avoid this error, you can upload
+your vector data to a public location(e.g., via Google Drive) and use it as an URL. The data stored in the URL can be loaded using 
+:py:meth:`~openeo.rest.connection.Connection.vectorcube_from_paths`
+
+The code-snippets shown below provides an example of how this can be achieved:
+
+.. code-block:: python
+
+    parcels = connection.vectorcube_from_paths(["https://artifactory.vgt.vito.be/testdata-public/parcels/32TPT.pq"],format="parquet")
```

Comment:
I would avoid putting a real VITO (artifactory) URL here, just use a dummy URL, like `https://example.com/parcels.pq`

Reply:
or maybe a real url on github? Maybe also working with a smaller file, so that it can be tested easily without running a very large job.